x509/cert_validations.go

```diff
@@ -10,11 +10,22 @@ import (
 	"time"
 )
 
+var kuLeafCertBlocked = x509.KeyUsageContentCommitment |
```

Comment:
One minor question here, the [certificate-requirements](https://github.com/notaryproject/notaryproject/blob/main/signature-specification.md#certificate-requirements) does not mention KeyUsageContentCommitment must be blocked for leaf certificates. Should we update the spec?

Reply:
Yep, will add it to cert requirement.